PS-palindrome-search/README.md

```diff
@@ -9,4 +9,7 @@
   - When there is a second palindrome => 'Found Palindrome: [PALINDROME]'
 - After you complete the exercise, provide any notes on your code below such as how to run your example
 
-## Candidate Notes:
\ No newline at end of file
+## Candidate Notes:
+   - First Longest Palindrome could be substrig for example "referrer". Here referrer is not a palindrome but 'refer' is. So the second longest palindrome is 'erre'. 
```

Comment:
Example should move before Candidate Notes section. Also, substring is mispelled